src/components/ResultDisplay.tsx

```diff
@@ -41,22 +56,82 @@ export function ResultDisplay({ result }: ResultDisplayProps) {
 			<table className={styles.estimates}>
 				<thead>
 					<tr>
-						<th className={styles.th}>Package Name</th>
-						<th className={styles.th}>Estimate</th>
-						<th className={styles.th}>Status</th>
+						<th className={styles.th}>
+							Package Name
```

Comment:
Can the text go inside the button too? I'm finding it surprising that clicking _Package Name_ doesn't trigger anything.

Reply:
Done! This was a little tricky, lots of unexpected side effects in the CSS. Let me know what you think! I am unsure about the bold styling for `isActive`, I think it's helpful to let the user know which column is selected. I played with a few options and this is what I thought looked best.